search-cache-pipeline.yml

```diff
@@ -0,0 +1,83 @@
+schedules:
+- cron: 0 * * * *
+  displayName: Run every hour
+  branches:
+    include:
+    - main
+  always: true
+
+pool:
+    name: Hosted Ubuntu 1604
+
+steps:
+- checkout: self
+
+- script: '$(Build.SourcesDirectory)/build.sh'
+
+- task: UseDotNet@2
+  displayName: 'Use .NET 3.1'
+  inputs:
+    packageType: sdk
+    version: 3.1.x
+    installationPath: $(Build.SourcesDirectory)/.dotnet
+
+- task: UseDotNet@2
+  displayName: 'Use .NET 5.0.100'
+  inputs:
+    packageType: sdk
+    version: 5.0.100
+    installationPath: $(Build.SourcesDirectory)/.dotnet
+
+- task: UseDotNet@2
+  displayName: 'Use .NET 5.0.300'
+  inputs:
+    packageType: sdk
+    version: 5.0.300
+    installationPath: $(Build.SourcesDirectory)/.dotnet
+
+- bash: >
+    curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin
+    -Version 6.0.100-rc.2.21420.30
```

Comment:
can we set up this version to be updated automatically with other runtime versions?

Reply:
Moved out to a new issue: https://github.com/dotnet/templating/issues/3757